content/HowItWorks/DeviceTypes.md

```diff
@@ -6,25 +6,31 @@ weight = 10
 
 In order to be certified a Matter device must conform to one of the approved device types.
 
-As of Matter 1.2 the following types are certifiable.
-
-- Lighting
+As of Matter 1.3 the following types are certifiable.
+
+- [Utility Device Types](#utility-device-types)
+  - [Root Node](#root-node)
+  - [Power Source](#power-source)
+  - [OTA Requestor](#ota-requestor)
+  - [OTA Provider](#ota-provider)
+  - [Bridged Node](#bridged-node)
+  - [Electrical Sensor](#electrical-sensor)
+  - [Device Energy Management](#device-energy-management)
+- [Application Device Types](#application-device-types)
   - [On/Off Light](#onoff-light)
   - [Dimmable Light](#dimmable-light)
   - [Color Temperature Light](#color-temperature-light)
   - [Extended Color Light](#extended-color-light)
-- Smart Plugs/Outlets and Other Actuators
```

Comment:
Is it intentional to remove these headings?

Reply:
No! - I think my editor did that - I'll revert.